doc/modules/clustering.rst

```diff
@@ -1078,17 +1168,17 @@ simplify the hyperparameter space.
 
 .. topic:: References:
```

Comment:
Should this block be collapsed as well?

Reply:
I guess the solution in #28401 will help avoid the breaking of cross-references, but for the moment it is maybe more safe not to hide cross-references inside dropdowns, WDYT?

Reply:
Sure, let's keep it as is :)